libs/libwebsockets/Makefile

```diff
@@ -55,24 +57,20 @@ define Package/libwebsockets-openssl
 	TITLE += (OpenSSL)
 	DEPENDS += +libopenssl
 	VARIANT:=openssl
-	CONFLICTS:=libwebsockets-full
 endef
 
 define Package/libwebsockets-mbedtls
 	$(call Package/$(PKG_NAME)/Default)
 	TITLE += (mbedTLS)
 	DEPENDS += +libmbedtls
 	VARIANT:=mbedtls
-	PROVIDES:=libwebsockets
-	CONFLICTS:=libwebsockets-openssl
```

Comment:
> remove conflicts as similar packages don't specify at all.

This is really not true at all. Did you figure out why there are conflicts and what is their purpose? It does not mean that those are not wrong. Please check this out: https://github.com/openwrt/packages/pull/19701

Reply:
IMO, package should avoid to require specify variant.

`mosquitto-ssl` requires `libwebsockets-openssl`, and `ttyd` requires `libwebsockets-full`.
If there's other package requires `libwebsockets`, don't care about the variants at all, then which they should install?

Of course, if it's the real case, then make `libwebsockets-full` provides `libwebsockets` and extra `libwebsockets-openssl`.

And if `libwebsockets-full` provides `libwebsockets-openssl`, then it's ok:
```
root@OpenWrt:~# opkg list-installed | grep -E '(libwebsockets|ttyd|mosquitto-ssl)'
libwebsockets-full - 4.3.3-1
mosquitto-ssl - 2.0.17-1
ttyd - 1.7.3-1
```